src/test/java/uk/gov/hmcts/reform/professionalapi/service/impl/OrganisationServiceImplTest.java

```diff
@@ -460,4 +461,23 @@ private void assertExpectedOrganisationResponse(OrganisationResponse organisatio
         assertThat(organisationResponse.getOrganisationIdentifier()).isNotNull();
         assertThat(organisationResponse.getOrganisationIdentifier().length()).isEqualTo(orgIdLength);
     }
+
+    @Test
+    public void testAddContactInformationToOrganisation() throws NoSuchMethodException, IllegalAccessException {
+        List<ContactInformationCreationRequest> contactInformationCreationRequest = contactInformationCreationRequests;
+        ContactInformationCreationRequest contactInformationCreationRequestMock = new ContactInformationCreationRequest("addressLine-1", "addressLine-2", "addressLine-3", "townCity", "county", "country", "postCode", dxAddressRequests);
+        contactInformationCreationRequest.add(contactInformationCreationRequestMock);
+        Organisation organisationMock1 = new Organisation("some-org-name", OrganisationStatus.ACTIVE, "PENDING", "Test", Boolean.TRUE, "Demo");
```

Comment:
Please remove "mock" from these variable names as they are not mocks, can be misleading to read otherwise

Reply:
Done